iocAdmin/Db/ioc_stats.db

```diff
@@ -0,0 +1,301 @@
+record(stringin, "$(IOCNAME):STARTTOD")
+{
```

Comment:
I'm seeing duplication of records here, can you explain if and how these are supposed to differ? For example there's an `$(IOCNAME):STARTTOD` record in the ioc.template file which the build instantiates into several different installed DB files.